code-studio/src/js/code-studio.js

```diff
@@ -16,6 +16,8 @@ require('./videos');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.ColorPicker = require('react-color').default;
```

Comment:
Actually, the following seemed to work for me - changes to package.json (I'm running into this for a different library we might use):
```
"browserify-shim": {
    "blockly": {
      "exports": "Blockly"
    },
    "react": "global:React"
  },